packages/mdc-dialog/package-lock.json

```diff
@@ -1,25 +1,25 @@
 {
-  "requires": true,
-  "lockfileVersion": 1,
-  "dependencies": {
-    "focus-trap": {
-      "version": "4.0.2",
-      "resolved": "https://registry.npmjs.org/focus-trap/-/focus-trap-4.0.2.tgz",
-      "integrity": "sha512-HtLjfAK7Hp2qbBtLS6wEznID1mPT+48ZnP2nkHzgjpL4kroYHg0CdqJ5cTXk+UO5znAxF5fRUkhdyfgrhh8Lzw==",
-      "requires": {
-        "tabbable": "^3.1.2",
-        "xtend": "^4.0.1"
-      }
-    },
-    "tabbable": {
-      "version": "3.1.2",
-      "resolved": "https://registry.npmjs.org/tabbable/-/tabbable-3.1.2.tgz",
-      "integrity": "sha512-wjB6puVXTYO0BSFtCmWQubA/KIn7Xvajw0x0l6eJUudMG/EAiJvIUnyNX6xO4NpGrJ16lbD0eUseB9WxW0vlpQ=="
-    },
-    "xtend": {
-      "version": "4.0.1",
-      "resolved": "https://registry.npmjs.org/xtend/-/xtend-4.0.1.tgz",
-      "integrity": "sha1-pcbVMr5lbiPbgg77lDofBJmNY68="
-    }
-  }
+	"requires": true,
```

Comment:
Remove this file from the PR, it looks like all that changed was indentation for some reason (and it changed to hard tabs, which I didn't think npm ever uses??)